pkg/template/helper.go

```diff
@@ -9,23 +9,28 @@ import (
 	"github.com/rancher/distros-test-framework/shared"
 )
 
-// upgradeVersion upgrades the version of RKE2 and updates the expected values
+// upgradeVersion upgrades the product version
 func upgradeVersion(template VersionTestTemplate, version string) error {
 	err := testcase.TestUpgradeClusterManually(version)
 	if err != nil {
 		return err
 	}
 
+	updateExpectedValue(template)
+
+	return nil
+}
+
+// updateExpectedValue updates the expected values
```

Comment:
i would just improve a little the comment here, like saying that it updates getting the values from flag ExpectedValueUpgrade